.github/workflows/pathogen-repo-build.yaml

```diff
@@ -252,12 +252,34 @@ jobs:
           ./bin/interpolate-env < ./text-templates/attach-aws-batch.md \
           > "$GITHUB_STEP_SUMMARY"
 
+      - if: always()
+        id: prefix-paths
+        name: Prefix artifact paths with ${{ env.BUILD_DIR }}/
+        env:
+          paths: ${{ inputs.artifact-paths }}
+        run: |
+          prefixed_paths=$(
+            yq -n '
+                strenv(paths)
+              | trim
+              | split("\n")
+              | .[] |= env(BUILD_DIR) + "/" + .
+              | join("\n")
+            '
+          )
```

Comment:
Note that this doesn't account for comments or exclusion patterns in `inputs.artifact-paths`, both of which [are supported](https://github.com/actions/toolkit/tree/main/packages/glob#patterns). Exclusions in particular seem useful to allow.

This prefixing business is kinda unfortunate accidental complexity. I wish we could avoid it by cd-ing to the expected build dir first, but it seems like we can't using `working-directory:` on steps with `uses:`, only `run:`.

I wonder about avoiding it another way, such as not checking out the build repo into a subdir but into the default working dir.

Reply:
Hmm, I guess we can switch the repo directories. The `nextstrain/.github` repo can be in a subdir while the pathogen build repo can be in the default working dir. I think the only downside(?) is the `.github` subdir would be included in a AWS Batch build. (This can be excluded once the CLI [supports exclusion patterns](https://github.com/nextstrain/cli/issues/219))

Reply:
Ah, I was thinking we'd clone the `nextstrain/.github` repo into somewhere outside of the working dir (i.e. outside of `$GITHUB_WORKSPACE`). Either hardcoded as `/tmp` or using `$TMP` or `$RUNNER_TEMP`. Last option is probably the best.

Reply:
I was trying to use `$RUNNER_TEMP` but annoyingly [`uses` path cannot be dynamic](https://github.com/orgs/community/discussions/25246) and must be hardcoded.

Reply:
Ah, well a hardcoded path outside of the working dir should still work, right? e.g. `/tmp/nextstrain/.github` or maybe `~/nextstrain/.github`?

Reply:
Hmm, this requires a bit of a workaround...

[`action/checkout` does not allow cloning outside of `GITHUB_WORKSPACE`](https://github.com/actions/checkout/issues/197). So I tried to `mv` the repo outside after checkout, but [`uses` still does not recognize the hardcoded /tmp/nextstrain/.github path](https://github.com/joverlee521/nextstrain-testing/actions/runs/5395356766). 

Then I tried to `mv` after running the local composite action, but [the workflow fails in the post job clean up steps](https://github.com/joverlee521/nextstrain-testing/actions/runs/5395548412/jobs/9798142872).

So the workflow will have to `mv` the `nextstrain/.github` repo out before the build step then `mv` it back to allow the clean up jobs to find the files again.

Reply:
Well that's unfortunate. Jeez. That sounds like more faffing about than we'd like.

Ok, so maybe let's just put the nextstrain/.github clone into a subdir of the working directory (with a name less conflict prone than `.github`)?

Until Nextstrain CLI [supports exclusions(https://github.com/nextstrain/cli/issues/219), we can a) accept it'll be part of the build dir and likely not cause issues or b) be sneaky and abuse the [currently hardcoded list of exclusions](https://github.com/nextstrain/cli/blob/6e36f6d7bdafcb984f62e2a9cc2a9cf9adf0be57/nextstrain/cli/runner/aws_batch/s3.py#L51-L66), e.g. clone it into `.git/nextstrain/.github/` or something...

Reply:
Oh, I didn't realize there was a hardcoded list of exclusions. I'll update to clone into `.git/nextstrain/.github`.

Reply:
Updated in https://github.com/nextstrain/.github/pull/44/commits/d0331939ec60d7a16c7259432cc0202db5536bbd